flags.yml

```diff
@@ -55,9 +55,33 @@
   default: false
   contact: Query Team
 
-- name: Push Down Window Aggregate Rest
-  description: Enable non-Count, non-Sum variants of PushDownWindowAggregateRule and PushDownWindowAggregateRule (stage 2)
-  key: pushDownWindowAggregateRest
+- name: Push Down Window Aggregate Min
```

Comment:
Can you add these flags to ConfigCat before merging. That way we won't spam the logs when this code gets deployed.